### DISCUSSION TITLE
How to display additional symbols in Aerial

### DISCUSSION THREAD

**Discussion by jorgesca** [LazyVim/LazyVim#3714] [Q&A]
I don't get the mechanism for overriding LazyVim defaults. I want to be able to display variables in Aerial when working with Python. I've looked in the documentation, searched for examples, but still don't manage to get this right. I've found that LazyVim defines a `kind_filter` option that is used to create `filter_kind` for Aerial. I tried extending `kind_filter` with a `python` option as shown below but I still get the default behavior:
```lua  
{
    "LazyVim/LazyVim",
    opts = {
      kind_filter = {
        python = {
          "Class",
          "Constant", -- add this
          "Constructor",
          "Enum",
          "Function",
          "Interface",
          "Module",
          "Method",
          "Struct",
          "Variable", -- add this
        },
      },
    },
  },
```
I have found an [issue](https://github.com/LazyVim/LazyVim/issues/2216) where this topic is touched, but as I said before, I am missing something to get this working.

**Comment by douglascamata** [comment]
@jorgesca I hope it's not too late, mate, but you can use the `lazy.lua` file and do something like this: 

```lua
LazyVim.config.kind_filter["go"] = false
```

I do this to have very complete LSP symbols for Go.
